ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -1292,6 +1381,53 @@ def test_cholesky(
     )
 
 
+# cov
+@handle_test(
+    fn_tree="functional.ivy.cov",
+    dtype_x1_x2_cov=_get_dtype_value1_value2_cov(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=2,
+        min_dim_size=2,
+        max_dim_size=5,
+        min_value=1,
+        max_value=1e10,
+        abs_smallest_val=0.01,
+        large_abs_safety_factor=2,
+        safety_factor_scale="log",
+    ),
+    test_gradients=st.just(False),
```

Comment:
If you remove the `out` argument, you should add `test_with_out=st.just(False),` here.